Dockerfile

```diff
@@ -21,13 +21,14 @@ ENV CC="gcc-12" \
 WORKDIR /tmp/tenzir
 
 COPY --from=fluent-bit-package /root/fluent-bit_*.deb /root/
-COPY scripts/debian/install-dev-dependencies.sh ./scripts/debian/
+COPY scripts/debian/install-aws-sdk.sh ./scripts/debian/
+RUN ./scripts/debian/install-aws-sdk.sh
```

Comment:
Does it make sense to move this into a separate target so that we don't always need to rebuild the AWS SDK when changing any line above these two?